learning/step_by_step/scripting.rst

```diff
@@ -7,75 +7,84 @@ Introduction
 ------------
 
 Much has been said about tools that allow users to create video games
-without programming. It's been a dream for many independent developers
-to create games without learning how to code. This need has been around
-for a long time, even inside companies, where game designers wish to
-have more control of the game flow.
-
-Many products have been shipped promising a no-programming environment,
-but the result is often incomplete, too complex or inefficient compared
-to traditional code. As a result, programming is here to stay for a long
-time. In fact, the general direction in game engines has been to add
-tools that try to reduce the amount of code that needs to be written for
-specific tasks, to speed up development.
-
-In that sense, Godot has taken some useful design decisions towards that
-goal. The first, and most important, is the scene system. The aim of it is
-not obvious at first, but is understandable later on. That is, it relieves
-programmers from the responsibility of designing and structuring code.
-
-When designing games using the scene system, the whole project is
-fragmented into *complementary* scenes (not individual ones). Scenes
-complement (i.e. help) each other, instead of being separate. There will be
-plenty of examples of this later on, but it's very important to remember it.
-
-For those with a good amount of programming expertise, this means a
```

Comment:
Note to reviewers: I deleted this section. Please confirm.

As a programmer with a decent amount of programming expertise, I would say that I would not think about using MVC in game dev at all. (The entity-component pattern is at the heart of many games instead, I think) Maybe for UI, but even then, there are plenty of other patterns as well, e.g. MVVM. I guess my thought was, I don't really see how this paragraph helps experienced devs, who might just get stuck here and argue about the details. Meanwhile inexperienced or non-devs may be totally unaware of what MVC even is and this would only serve to distract them. 

Overall, I think this paragraph isn't necessary because I instead emphasized:

"... fundamentally [Godot's] goal is to relieve programmers from the responsibility of having to implement an overall architecture." 

which I believe captures the spirit of the original text. What do you think?